docs/untitled-test.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Does this test page need to be committed / included?

Reply:
No, it can be removed. But I think it would be good to keep a set of regression tests and corner cases somewhere...

Reply:
That's a great idea. What if we created a hidden section called "Test" that had a child page for each test case? You've done a fair amount of these tests already, do you think it'd be hard to compile them?

```
|-- Tests
    |-- test-case 1
    |-- test-case 2
```

Reply:
I'll be happy to do that, but it can surely wait until after 0.3.2.

However, I don't see how to test navigation if the section is permanently hidden, and all the `nav_exclude` fields would change the source files. Perhaps the simplest would be to add `Tests/` to `excludes` in `_config.yml`, so the test cases aren't even processed unless that setting is overridden?